modules/vba_documents/spec/sidekiq/upload_scanner_spec.rb

```diff
@@ -46,11 +46,11 @@
 
       it 'logs that the upload status progressed to "uploaded"' do
         with_settings(Settings.vba_documents.s3, enabled: true) do
-          log_message = 'VBADocuments: UploadSubmission progressed to "uploaded" status'
-          log_details = { 'job' => described_class.name }.merge(upload.as_json)
-
           described_class.new.perform
 
+          log_message = 'VBADocuments: UploadSubmission progressed to "uploaded" status'
+          log_details = { 'job' => described_class.name }.merge(upload.reload.as_json)
```

Comment:
Had to define these after `described_class.new.perform` and use `.reload` since the `UploadSubmission` is modified mid-job.